pkg/cmd/eventlistener/describe.go

```diff
@@ -179,6 +182,25 @@ func describeEventListenerOutput(w io.Writer, p cli.Params, f *cliopts.PrintFlag
 	return printer.PrintObject(w, el, f)
 }
 
+func describeEventListenerOutputURL(w io.Writer, p cli.Params, name string) error {
+	cs, err := p.Clients()
+	if err != nil {
+		return err
+	}
+
+	el, err := cs.Triggers.TriggersV1alpha1().EventListeners(p.Namespace()).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	if getURL(*el) == "" {
+		fmt.Fprintf(w, "%s\n", "URL of the eventlistener is not available yet")
```

Comment:
This should probably be an error:

```suggestion
		fmt.Errorf("url of EventListener %s not available yet", el.Name)
```

Reply:
It might also make sense to add this error return within `getURL` to make the func a bit more reusable/it would help standardize the error message for this case.

Reply:
@danielhelfand  Thanks for the review, adding error return within `getURL`  will create issues as `getURL` is used at multiple places for e.g  formatting for `describe` so is it fine if we keep the `getURL` function with just returning an empty string